ast/ddl.go

```diff
@@ -3426,3 +3449,68 @@ func (n *FlashBackTableStmt) Accept(v Visitor) (Node, bool) {
 	}
 	return v.Leave(n)
 }
+
+type PlacementActionType int
+
+const (
+	PlacementAdd PlacementActionType = iota + 1
+)
+
+type PlacementRole int
+
+const (
+	PlacementRoleFollower PlacementRole = iota + 1
+	PlacementRoleLeader
+	PlacementRoleLearner
+	PlacementRoleVoter
```

Comment:
May this order would be more natural:
```suggestion
	PlacementRoleLeader PlacementRole = iota + 1
	PlacementRoleFollower
	PlacementRoleLearner
	PlacementRoleVoter
```